Library/Formula/joshua-v.rb

```diff
@@ -0,0 +1,55 @@
+class JoshuaV < Formula
+  desc "Joshua is an open-source statistical machine translation decoder."
+  homepage "http://joshua-decoder.org/"
+  url "http://cs.jhu.edu/~post/files/joshua-v6.0.4.tgz"
+  sha256 "0cf6674dc73f30e49a22179e90182492f8e8eea46710b505edb9acfb4d953d9a" 
```

Comment:
This was the original checksum I calculated and was asked to lowercase everything. Which format o you want? I can update the PR to accomodate if you prefer.